ci/ray_ci/tests.env.Dockerfile

```diff
@@ -4,6 +4,7 @@ ARG BASE_IMAGE
 FROM "$BASE_IMAGE"
 
 ARG BUILD_TYPE
+ARG BUILDKITE_PIPELINE_ID
```

Comment:
this will force each pipeline to have their own caches, because the arg value will be captured.

Reply:
I guess this is not a wanda docker build file, so it is okay?

Reply:
yes, this last layer is not cachable since it also installs ray